src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -417,7 +416,6 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_REGISTER_PASS_X64(manager, ConvertToInteraction);
     CPU_REGISTER_PASS_X64(manager, ConvertInteractionInt8);
     CPU_REGISTER_PASS_ARM(manager, ConvertReduceMultiAxis);
-    CPU_REGISTER_PASS_ARM(manager, MishDecomposition);
```

Comment:
This change should break 32bit ARM there JIT is not available

Reply:
> This change should break 32bit ARM there JIT is not available

was fixed, thanks